packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -120,6 +120,26 @@ function unmemoType(type) {
   return isMemo(type) ? type.type : type;
 }
 
+function transformSuspenseElement(el, { suspenseFallback }) {
+  let { children } = el.props;
+
+  if (suspenseFallback) {
+    const { fallback } = el.props;
+    children = replaceLazyWithFallback(children, fallback);
+  }
+
+  const FakeSuspenseWrapper = (props) => React.createElement(el.type, { ...el.props, ...props }, children);
```

Comment:
this seems like it might break class components that use private fields, or functional components that use hooks.

Reply:
In all cases `el.type` is `react.Suspense`, the wrapper of `lazy` and `fallback` inside. I just create it after changed children inside.

Reply:
Perhaps it would be cleaner to inline `transformSuspenseElement`, since it's only used once, and then it couldn't possibly be invoked when `el` isn't a Suspense element?

Reply:
I updated it, make inline `transformSuspenseElement` function. Thanks for feedback!